newscoop/src/Newscoop/ArticlesBundle/Resources/public/app/comments-ctrl.js

```diff
@@ -232,7 +232,8 @@ angular.module('editorialCommentsApp').controller('EditorialCommentsCtrl', [
 	        	$scope.textareaMessage = {};
 	        	$scope.textareaReply = {};
 	        	comment.isReplying = false;
-	        	$scope.enableTyping = false
+	        	$scope.enableTyping = false;
+                $scope.stopRefreshing = false;
```

Comment:
in Sublime - View -> indentation -> convert indentation to spaces

convert indentations to spaces. and mark also - indent using spaces

you can make it as default in sublime config